src/iwant_bot/pool.py

```diff
@@ -20,6 +21,7 @@ def __init__(self, storage):
         self._time_relevant_requests = set()
         self._blacklisted_requests = set()
         self._time_conflicting_requests = set()
+        self.users = IgnoreList()
```

Comment:
I would not guess that the attribute `users` has an `IgnoreList` instance assigned to it. Why not to call it `ignore_list`?